pkg/kubelet/pod_workers.go

```diff
@@ -180,7 +180,13 @@ func (p *podWorkers) managePodLoop(podUpdates <-chan UpdatePodOptions) {
 		}
 		if err != nil {
 			glog.Errorf("Error syncing pod %s (%q), skipping: %v", update.Pod.UID, format.Pod(update.Pod), err)
-			p.recorder.Eventf(update.Pod, v1.EventTypeWarning, events.FailedSync, "Error syncing pod, skipping: %v", err)
+			// if we failed sync, we throw more specific events for why it happened.
```

Comment:
@yujuhong - i am open to adding an error when the network is not ready or any other more specific error situation.  what i dislike is that we used FailedSync as a catch-all and as a result we duplicated the same information twice.  As a user, what does FailedSync even mean?  i would argue it only has meaning for kubelet authors, and not kubernetes users.  if i got an event "NetworkNotReady" with a message that is way better than "FailedSync".